src/hooks/wallet/__tests__/useSpendBundle.reroute.test.tsx

```
@@ -0,0 +1,164 @@
+    rainApi: {
+        prepareWithdrawal: (...args: unknown[]) => mockPrepareWithdrawal(...args),
+        submitWithdrawal: (...args: unknown[]) => mockSubmitWithdrawal(...args),
+        stampWithdrawal: jest.fn().mockResolvedValue(undefined),
+    },
+}))
+jest.mock('@/utils/balance.utils', () => ({ usdcUnitsToRainCents: (x: bigint) => x }))
+jest.mock('@/app/actions/clients', () => ({
+    peanutPublicClient: { readContract: (...args: unknown[]) => mockReadContract(...args) },
+}))
+
+// eslint-disable-next-line import/first -- mocks must register before import
```

ESLint

[error] Definition for rule 'import/first' was not found.

(import/first)